spec/models/gamesession_spec.rb

```diff
@@ -1,5 +1,20 @@
 require 'rails_helper'
 
-RSpec.describe Gamesession, type: :model do
-  # pending "add some examples to (or delete) #{__FILE__}"
+RSpec.describe GameSession, type: :model do
+  subject { GameSession.new }
+  before { subject.save }
+  it 'word_to_guess should be assigned' do
+    expect(subject.word_to_guess).to_not eq(nil)
+  end
+
+  it 'should have an array list to select from' do
+    defined? GameSession::WORDS
+  end
+  it 'should select word_to_guess from the words array' do
+    expect(GameSession::WORDS).to include(subject.word_to_guess)
```

Comment:
I definitely feel like convention is to do this the other way around. 
```
expect(thing_to_be_tested).to be(a_value)
```

But I can't for the life of me think how this would work with the built in rspec matchers.